### ISSUE TITLE
v0.10.0 release does not build

### ISSUE THREAD

**Issue by messense** [guoxbin/dtool#22]
rustc version: rustc 1.45.2 (d3fb005a3 2020-07-31)

```
error[E0405]: cannot find trait `PrehashSignature` in crate `signature`
  --> /Users/lusheng.lls/.cargo/registry/src/github.com-1ecc6299db9ec823/signatory-secp256k1-0.17.0/src/lib.rs:25:10
   |
25 | #[derive(Signer)]
   |          ^^^^^^ not found in `signature`
   |
   = note: this error originates in a derive macro (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0405]: cannot find trait `PrehashSignature` in crate `signature`
  --> /Users/lusheng.lls/.cargo/registry/src/github.com-1ecc6299db9ec823/signatory-secp256k1-0.17.0/src/lib.rs:80:39
   |
80 | #[derive(Clone, Debug, Eq, PartialEq, Verifier)]
   |                                       ^^^^^^^^ not found in `signature`
   |
   = note: this error originates in a derive macro (in Nightly builds, run with -Z macro-backtrace for more info)

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0405`.
error: could not compile `signatory-secp256k1`.

To learn more, run the command again with --verbose.
warning: build failed, waiting for other jobs to finish...
error: failed to compile `dtool v0.10.0`, intermediate artifacts can be found at `/var/folders/8d/h3lyjgz14296j_lw7chgf5hc0000gp/T/cargo-installyB8mOX`
```

Maybe it needs to update lockfile for `signatory-secp256k1`?

**Comment by StevenBlack** [comment]
Confirming.  `cargo install dtool`, as in the readme, fails to build with the errors above ⬆️⬆️⬆️.

**Comment by guoxbin** [comment]
Fixed in v0.10.1 @messense @StevenBlack

**Comment by messense** [comment]
Thanks!
